lib/resources/r4/allergy_intolerance.yaml

```diff
@@ -113,18 +99,31 @@ fields:
   cardinality: 0..*
   type: BackboneElement
   description: Adverse Reaction Events linked to exposure to substance
-  url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.code
+  url: https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.code
   children:
 
   - name: manifestation
     required: 'Yes'
     cardinality: 1..*
     type: CodeableConcept
     description: Clinical symptoms/signs associated with the Event
-    url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.reaction.manifestation
+    url: https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance.reaction.manifestation
     binding:
       description: Clinical symptoms and/or signs that are observed or associated with the adverse reaction event.
       terminology:
```

Comment:
As per this JIRA https://jira2.cerner.com/browse/OPENSVC-8392, manifestation also supports proprietary codes.  Are we not supporting proprietary codes for manifestation anymore?

Reply:
No I haven't made any changes to support proprietary codes as it's not part of the Uplift work. May be an enhancement later.

Reply:
+1